website/pages/docs/internals/telemetry.mdx

```diff
@@ -162,6 +162,9 @@ These metrics cover measurement of token, identity, and lease operations, and co
 | `vault.expire.register`                   | Time taken for register operations                                          | ms     | summary |
 | `vault.expire.register-auth`              | Time taken for register authentication operations which create lease entries without lease ID           | ms       | summary |
 | `vault.identity.num_entities`             | Number of identity entities stored in Vault                                                             | entities | gauge   |
+| `vault.identity.entity.active.monthly` (cluster, namespace) | Number of distinct entities that created a token during the past month, per namespace. Only available if client count is enabled. Reported at the start of each month. | entities | gauge   |
```

Comment:
```suggestion
| `vault.identity.entity.active.monthly` (cluster, namespace) | Number of distinct entities that created a token during the past month, per namespace. Only available if client count is enabled. Reported at the start of each month. | entities | gauge |
```

Reply:
This file got moved so I can't apply the suggestion as-is, I tried to make the spacing consistent in the new file.